### ISSUE TITLE
Command to set Reminder

### ISSUE THREAD

**Issue by igwigg** [NiklasMM/sixgun_discord_bot#2]
eg.
`!setreminderfor <discordID/handle>, <Date>, <time>, <reason>`
`!remindme <date>, <time>, <reason>`
output may be in a defined channel at the specified date and time: @discordID don't forget <reason> or output as a direct message.

**Comment by NiklasMM** [comment]
Good idea, but should be limited to the users own account otherwise it would open the door to spamming.

**Comment by davidmn** [comment]
Users own account and Fab, open the spamming door just a crack
